landlab/components/landslides/landslide_probability.py

```diff
@@ -0,0 +1,535 @@
+            ('critical angle just before failure' +
+             ' due to friction between particles'),
+        'soil__density': 'wet bulk density of soil',
+        'soil__thickness': 'soil depth to restrictive layer',
+        'soil__mean_relative_wetness':
+            ('Indicator of soil wetness;' +
+             ' relative depth perched water table' +
+             ' within the soil layer'),
+        'landslide__probability_of_failure':
+            ('number of times FS is <=1 out of number of' +
+             ' iterations user selected'),
+        }
+
+# Run Component
+    @use_file_name_or_kwds
+    def __init__(self, grid, number_of_iterations=250,
+                 groundwater__recharge_distribution='uniform',
+                 groundwater__recharge_min_value=20.,
+                 groundwater__recharge_max_value=120.,
+                 groundwater__recharge_mean=None,
+                 groundwater__recharge_standard_deviation=None,
+                 groundwater__recharge_HSD_inputs=[],
+                 seed=0, **kwds):
+
+        """
+        Parameters
+        ----------
+        grid: RasterModelGrid
+            A grid.
+        number_of_iterations: int, optional
+            number of iterations to run Monte Carlo simulation (None)
+        groundwater__recharge_distribution: str, optional
+            single word indicating recharge distribution, either 'uniform',
+            'lognormal', 'lognormal_spatial,' or 'data_driven_spatial' (None)
+        groundwater__recharge_min_value: float, optional
+            minium groundwater recharge for 'uniform' (mm/d)
+        groundwater__recharge_max_value: float, optional
+            maximum groundwater recharge for 'uniform' (mm/d)
+        groundwater__recharge_mean: float, optional
+            mean grounwater recharge for 'lognormal' (mm/d)
+        groundwater__recharge_standard_deviation: float, optional
+            standard deviation of grounwater recharge for 'lognormal' (mm/d)
+        groundwater__recharge_HSD_inputs: list, optional
+            list of 3 dictionaries in order - HSD_dict {Hydroligic Source
+            Domain (HSD) keys: recharge numpy array values}, {node IDs keys:
+            list of HSD_Id values}, HSD_fractions {node IDS keys: list of
+            HSD fractions values} (none)
+        g: float, optional
+            acceleration due to gravity (m/sec^2)
+        seed: int, optional
+            seed for random number generation. if seed is assigned any value
+            other than the default value of zero, it will create different
+            sequence. to create a certain sequence repititively, use the same
+            value as input for seed.
+        """
+
+        # Initialize seeded random number generation        
+        self.seed_generator(seed)
+
+        # Store grid and parameters and do unit conversions
+        self._grid = grid
+        self.n = int(number_of_iterations)
+        self.g = 9.81
```

Comment:
We are using 'self.g' instead of 'self._g'. We have been consistently ignoring underscores. Should this be changed? If it is a must, we will invest time for that. If not, we would like to leave the variables, as they are.